### PR TITLE
Clean cache before upgrading packages

### PR DESCRIPTION
**Pull request by dangel101**
Bug-Url: https://bugzilla.redhat.com/show_bug.cgi?id=2104408

### CODE REVIEW

**Review by sandrobonazzola — APPROVED** [review]
LGTM

**Review by didib — APPROVED** [review]
Do we allow/document how to supply/set custom ansible vars? If so, it should be very easy to add 'when: not yum_do_not_clean_packages | bool' or something like that.

File: packaging/ansible-runner-service-project/project/roles/ovirt-host-upgrade/tasks/main.yml [hunk 1]
```diff
@@ -41,6 +41,10 @@
   set_fact:
     previous_vdsm_package: "{{ ansible_facts.packages['vdsm'] | first }}"
 
+- name: Clean yum cache
+  command:
+    cmd: yum clean packages
+
```
Comment (mwperina) [inline packaging/ansible-runner-service-project/project/roles/ovirt-host-upgrade/tasks/main.yml:47]: Do we want to abort upgrade if `yum clean packages` returns non-zero status? Or wouldn't it be better to continue and hope for the best?

**Comment by mwperina** [comment]
> Do we allow/document how to supply/set custom ansible vars? If so, it should be very easy to add 'when: not yum_do_not_clean_packages | bool' or something like that.

No, we don't have support for that, but I don't think it's worth to add a generic support for that

**Review by michalskrivanek — APPROVED** [review]

**Pull request merged**